prime-router/settings/STLTs/WA/wa-phd.yml

```diff
@@ -0,0 +1,119 @@
+      qualityFilter:
+              # "doesNotMatch(patient_city,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city.lower() != 'unknown')"
+              # "doesNotMatch(patient_first_name,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).name.given.lower() != 'unknown'"
+              # "doesNotMatch(patient_last_name,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).name.family.lower() != unknown"
+              # "doesNotMatch(patient_state,Unknown, unknown, UNKNOWN)"
+              - "Bundle.entry.resource.ofType(Patient).where(address.state.lower() != 'unknown')"
+              # "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
+              - "(Bundle.entry.resource.ofType(Patient).address.line.empty().not() or Bundle.entry.resource.ofType(Patient).address.postalCode.empty().not() or Bundle.entry.resource.ofType(Patient).telecom.empty().not())"
+              # "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)"
+              - "Bundle.entry.resource.ofType(DiagnosticReport).where( basedOn.resolve().extension('https://reportstream.cdc.gov/fhir/StructureDefinition/effective-date').empty() and specimen.resolve().collection.collected.empty() and effective.empty() and issued.empty() and result.resolve().effective.empty()).count() = 0"
+              # "hasValidDataFor(message_id,equipment_model_name,specimen_type,test_result,patient_last_name,patient_first_name,patient_dob,patient_city,patient_state)"
+              - "Bundle.identifier.value.empty().not()"
+              - "Bundle.entry.resource.ofType(DiagnosticReport).result.resolve().where(method.empty() or value.coding.code.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Patient).where(name.family.empty() or name.given.empty() or birthDate.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Patient).where(address.state.empty()).count() = 0"
+              # "isValidCLIA(testing_lab_clia,reporting_facility_clia)"
+              - "Bundle.entry.resource.ofType(ServiceRequest).performer.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+              - "Bundle.entry.resource.ofType(ServiceRequest).requester.resolve().organization.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+      routingFilter:
+      #        # "matches(abnormal_flag, A)"
+              - "Bundle.entry.resource.ofType(Observation).interpretation.coding.code = 'A'"
```

Comment:
This one actually needs to be moved to the condition filter. It looks like they don't want any negative results and to filter them out would need to be in the condition filter

Reply:
Done :)